README.rst

```diff
@@ -360,6 +360,10 @@ Choose from the list of available fixers:
                         Remove trailing commas in list
                         function calls.
 
+* **lower_case_cast** [symfony]
```

Comment:
maybe better `lowercase_casts` to match `lowercase_constants` and `lowercase_keywords`?